hw3.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+// Task 1 
+// Создайте переменные и присвойте им значения:
+
+const one = 'true';
+const two = false;
+const three = 17;
+const four = undefined;
+const five = null;
+
+console.log(typeof one);
+console.log(typeof two);
+console.log(typeof three);
+console.log(typeof four);
+console.log(typeof five);
+
+
+// Task 2
+//С помощью условий выведите в консоль разработчика наибольшее число.
+
+    let height = 15
+    let width = 20
+
+     if (height<width){
+         console.log(width);
+        }
+         else{
+            console.log(height);
+         }
+
+//Task 3 
+//Написать перебор от 1 до 20, где выведутся. все числа кратные трём.
+// Для вычисления кратности обратите внимание на оператор
+
+
+ for(let i = 1; i < 20; i++) {
+    if (i % 3) {
+         continue;
+     }
+     console.log(i);
+ }
```

Comment:
for(let i = 1; i < 20; i++) {
    if (i % 3 === 0) {
         console.log(i);
     }
 }